bin/verify-doc-links

```diff
@@ -19,12 +18,30 @@ glob("./{enterprise/,}frontend/src/**/*.{js,jsx}", (err, files) => {
       if (fs.existsSync(path)) {
         console.log(`Exists:  ${path}`);
       } else {
-        console.log(`Missing: ${path}`);
+        console.log(chalk.red(`Missing: ${path}`));
```

Comment:
Oh, and it's much easier to see the missing links now
![image](https://user-images.githubusercontent.com/31325167/220368333-0ac8fc3f-0f84-4430-8be2-34db9583ab22.png)